index.js

```diff
@@ -54,7 +79,26 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let averageIMDBRating = 0;
+  let counter = 0;
+  let ratings;
+
+  if (movies.length < 1) {
+    return averageIMDBRating;
+  } else {
+    for (let i = 0; i < movies.length; i++) {
+      for (let j = 0; j < movies[i].ratings.length; j++) {
+        ratings = parseFloat(movies[i].ratings[0].value);
+        averageIMDBRating += ratings;
+        counter++;
+      }
+    }
+  }
+  averageIMDBRating = (averageIMDBRating / counter) / 10;
```

Comment:
Did we need to divide this by 10? also it seems we're using `counter` to keep track of our movies. Couldn't we just use our `movies` array `.length` property